README.md

```diff
@@ -110,6 +110,35 @@ If you are looking for the codebase of the current production version of GATK, p
   ./gatk-launch PrintReadsSpark -I input.bam -O output.bam
   ```
 
+####Running GATK4 with inputs on Google Cloud Storage:
+
+* GATK can read BAM or VCF inputs from a Google Cloud Storage bucket. Just use the "gs://" prefix:
+  ```
+  ./gatk-launch PrintReads -I gs://mybucket/path/to/my.bam -L 1:10000-20000 -O output.bam
+  ```
+* For public datasets, that will just work. For private datasets you must set up authentication.
+* Option (a): run in a Google Cloud Engine VM
+  * If you are running in a Google VM then your credentials are already in the VM and will be picked up by GATK, you don't need to do anything special.
+* Option (b): use your own account
+  * Install [Google Cloud SDK](https://cloud.google.com/sdk/)
+  * Log into you account:
+  ```
+  gcloud auth application-default login
```

Comment:
If you type `gcloud auth login` (without the 'application-default`), will it still work?